client/.eslintcache

```diff
@@ -0,0 +1 @@
ng\\PalomaToSeries\\client\\web\\config\\webpack\\webpack-scopeHoisting.js","messages":[],"errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0}},"C:\\TFS\\PalomaScaffolding\\PalomaToSeries\\client\\web\\config\\webpack\\webpack-setEnvironmentVariable.js":{"size":212,"mtime":1511535258229,"hashOfConfig":"2gdddy","results":{"filePath":"C:\\TFS\\PalomaScaffolding\\PalomaToSeries\\client\\web\\config\\webpack\\webpack-setEnvironmentVariable.js","messages":[],"errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0}},"C:\\TFS\\PalomaScaffolding\\PalomaToSeries\\client\\web\\config\\webpack\\webpack-useExternalScripts.js":{"size":150,"mtime":1511535258230,"hashOfConfig":"2gdddy","results":{"filePath":"C:\\TFS\\PalomaScaffolding\\PalomaToSeries\\client\\web\\config\\webpack\\webpack-useExternalScripts.js","messages":[],"errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0}},"C:\\TFS\\PalomaScaffolding\\PalomaToSeries\\client\\web\\config\\webpack\\webpack-useHashedModuleIds.js":{"size":125,"mtime":1511535258231,"hashOfConfig":"2gdddy","results":{"filePath":"C:\\TFS\\PalomaScaffolding\\PalomaToSeries\\client\\web\\config\\webpack\\webpack-useHashedModuleIds.js","messages":[],"errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0}},"C:\\TFS\\PalomaScaffolding\\PalomaToSeries\\client\\web\\config\\webpack\\webpack-useHtmlTemplate.js":{"size":198,"mtime":1511535258232,"hashOfConfig":"2gdddy","results":{"filePath":"C:\\TFS\\PalomaScaffolding\\PalomaToSeries\\client\\web\\config\\webpack\\webpack-useHtmlTemplate.js","messages":[],"errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0}},"C:\\TFS\\PalomaScaffolding\\PalomaToSeries\\client\\web\\jest.config.js":{"size":289,"mtime":1511535258240,"hashOfConfig":"2gdddy","results":{"filePath":"C:\\TFS\\PalomaScaffolding\\PalomaToSeries\\client\\web\\jest.config.js","messages":[],"errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0}},"C:\\TFS\\PalomaScaffolding\\PalomaToSeries\\client\\web\\webpack.config.js":{"size":4562,"mtime":1511852678099,"hashOfConfig":"2gdddy","results":{"filePath":"C:\\TFS\\PalomaScaffolding\\PalomaToSeries\\client\\web\\webpack.config.js","messages":[],"errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0}},"C:\\TFS\\PalomaScaffolding\\PalomaToSeries\\client\\mobile\\AppEntry.js":{"size":86,"mtime":1511767288783,"hashOfConfig":"2gdddy","results":{"filePath":"C:\\TFS\\PalomaScaffolding\\PalomaToSeries\\client\\mobile\\AppEntry.js","messages":[],"errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0}},"C:\\TFS\\PalomaScaffolding\\PalomaToSeries\\client\\mobile\\App.js":{"size":449,"mtime":1511853806170,"hashOfConfig":"2gdddy","results":{"filePath":"C:\\TFS\\PalomaScaffolding\\PalomaToSeries\\client\\mobile\\App.js","messages":[],"errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0}}}
```

Comment:
Remove this file

Reply:
Me lo genera automaticamente cada vez que ejecuto desde comandos el linter, lo pondré en el .gitignore